tests/.suitespec.json

```diff
@@ -608,6 +608,7 @@
             "@django",
             "@dbapi",
             "@pg",
+            "tests/appsec/*",
```

Comment:
🤔  Why would appsec test sources affect some other component? Isn't the dependency on the appsec component itself enough?

Reply:
Ah I think the answer is in https://github.com/DataDog/dd-trace-py/pull/6819#pullrequestreview-1607084399